hugegraph-dist/src/assembly/descriptor/assembly.xml

```diff
@@ -51,10 +51,6 @@
         <fileSet>
             <directory>${top.level.dir}/hugegraph-dist/release-docs/</directory>
             <outputDirectory>/</outputDirectory>
-            <includes>
-                <include>LICENSE*</include>
-                <include>NOTICE*</include>
-            </includes>
```

Comment:
remember to check it in binary release file (could use [validate action](https://github.com/apache/incubator-hugegraph/actions/workflows/validate-release.yml) after we publish it to svn)

![image](https://user-images.githubusercontent.com/17706099/216114086-ad3e6b38-bb36-488b-ac5e-9a75b43e03c3.png)